build.gradle

```diff
@@ -6,11 +6,11 @@
  * user guide available at https://docs.gradle.org/3.3/userguide/tutorial_java_projects.html
  */
 
-// Adding these plugins tells Gradle we want
-// tasks for building, running, testing, and
-// deploying Java applications.
 plugins {
+  // Apply the java plugin to add support for Java
   id 'java'
+
+  // Apply the application plugin to add support for building a CLI application
```

Comment:
Nice change in the comment – thanks. I don't often see good/interesting examples of comments, and even less so for _changes_ in comments, but this change definitely adds value in a useful way.